### DISCUSSION TITLE
关于停贷的操作流程，有人总结一下吗？

### DISCUSSION THREAD

**Discussion by xiaohao587** [WeNeedHome/SummaryOfLoanSuspension#1168] [General]
现在只看到了统计全国烂尾楼和停贷告知书的情况，有没有大佬把合法停贷的操作流程给梳理总结一下，大致就两种情况把！
1、未到交房日期，但工地处于停工、表演式复工、施工进度和交房日期不匹配情况，应如何停贷？
2、已到交房日期，未如期交房，应如何停贷？
告知书发到网上，还需要通过什么方式发给哪些单位，如何向银行申请，应该注意什么？有大佬给总结一下吗？！

恒大的问题某党还愿不愿意解决啊！

**Comment by Cegazhos** [comment]
最好问律师。你必应也能搜到。

**Comment by cccp057** [comment]
不存在合法停贷，停贷一定是非法的。
关键是有多少人能做到停贷，做不到自然一切免谈

**Reply by xiejun946** [reply]
没有合法路径，就是破罐子破摔，赌上征信
